meson.build

```diff
@@ -23,7 +26,7 @@ c_args = [
 ]
 
 subdir('data')
-# subdir('po')
```

Comment:
Very tiny thing, but I think you don't have to remove this line, but just remove `#` in the beginning of this line.
In the [Getting Started Doc](https://elementary.io/ja/docs/code/getting-started#Adding-Translations), it says to add `subdir('po')` at the last line, but I think it's better to stay it here because other `subdir` is declared here.

Reply:
Thanks for everything. I will see what i can do to fix these.